docs/DeveloperQuickStart.md

```diff
@@ -42,16 +46,27 @@ $ $GOPATH/bin/rmd
 
 Bash shell script `hacking.sh` checks coding style using `go fmt` and `golint`.
 
-Before you commit your changes, run `./hacking.sh` and address errors before you push your changes.
+Before you commit your changes, run `hacking.sh` in scripts directory,
+and address errors before you push your changes.
```

Comment:
Add more description about using hacking, e.g. hacking.sh -f to do a full code checking

Reply:
Fixed.